### DISCUSSION TITLE
Creating stripe products

### DISCUSSION THREAD

**Discussion by DjamelAbbou** [flutter-stripe/flutter_stripe#1449] [General]
Hi everyone, hope you are doing great.

I wanted to ask if this library has an API that can be used to create stripe products ?
For example on nodejs i can do the following to create a stripe product: stripe.products.create({...})

**Comment by remonh87** [comment]
hey this sdk only handles collecting payment data from the frontend. I would recommend to checkout the official stripe docs: https://stripe.com/docs/invoicing/products-prices
